### PR TITLE
Initial README

### PR DESCRIPTION
**Pull request by kerkhofsd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,115 @@
...
+* `eu.xenit.alfresco.healthprocessor.task.delay=0`  
+  An optional delay before the processor starts.
+* `eu.xenit.alfresco.healthprocessor.processing.node-batch-size=1000`  
+  Node batch size of the processing. This is the size of the batch that will be requested from the `IndexingStategy`
+  and the size of the batches offered to the `HealthProcessorPlugin` implementations
+* `eu.xenit.alfresco.healthprocessor.processing.max-batches-per-second=-1`
+  Optional rate limitation to minimize load on the system. This number indicates the maximum number of batches each
+  plugin may process each second. A number smaller than or equal to 0 indicates no rate limiting is applied.
+* `eu.xenit.alfresco.healthprocessor.processing.read-only=true`  
+  The Health Processor wraps the processing of each bath for each plugin in a new transaction. This property configures
```
Comment (driesdeproost) [inline README.md:73]: typo:

`bath` instead of `batch`

File: README.md [hunk 2]
```diff
@@ -3,14 +3,115 @@
...
+
+## Out of the box functionality
+
+Next to the core framework, this module contains a list of useful tools that contain basic integrity check for your
+Alfresco repository. Please note that all available plugins and reporters are disabled by default and should be enabled
+as desired.
+
+### IndexingStrategy implementations
+
+The Health Processor uses a single indexing strategy to iterate of (a subset of) nodes in Alfresco. The active strategy
```
Comment (driesdeproost) [inline README.md:87]: `to iterate over`?

File: README.md [hunk 3]
```diff
@@ -20,4 +121,66 @@ eu.xenit.alfresco.healthprocessor.plugin.content-validation.properties=cm:conten
...
+```xml
+
+<dependency>
+    <groupId>eu.xenit.alfresco</groupId>
+    <artifactId>alfresco-health-processor-platform</artifactId>
+    <version>${last-version}</version>
+</dependency>
+```
+
+Please notice that the HealthProcessor is implemented an Alfresco subsystem. Therefore you can choose to make your
```
Comment (driesdeproost) [inline README.md:171]: `as an Alfresco subsystem`?

File: README.md [hunk 4]
```diff
@@ -20,4 +121,66 @@ eu.xenit.alfresco.healthprocessor.plugin.content-validation.properties=cm:conten
...
+customizations available in the main Alfresco Spring context or as an extension of the HealthProcessor Spring context.
+For the former, configuration files (XML / properties) should be available in:
+`classpath:alfresco/extension/subsystems/HealthProcessor/default/default`.
```
Comment (driesdeproost) [inline README.md:174]: folder structure contains `default` two times, intended?

**Comment by driesdeproost** [comment]
nice :+1:

**Comment by driesdeproost** [comment]
nice :+1:

**Review by driesdeproost — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (kerkhofsd) [reply README.md:174]: Yes. See: https://hub.alfresco.com/t5/alfresco-content-services-hub/alfresco-subsystems/ba-p/289509

**Review by thijslemmens — APPROVED** [review]

**Pull request merged**